test/Array_test.jl

```diff
@@ -160,6 +160,22 @@
     end
 
     @testset "Correctness" begin
+        @testset "Rx" begin
+            @test begin
+                g = Rx(1, θ = 0)
+                Matrix(g) ≈ LinearAlgebra.I(2^length(g))
+            end
+
+            @test begin
+                g = Rx(1, θ = π / 2)
+                Matrix(g) ≈ 1 / sqrt(2) * [1 -1im; -1im 1]
+            end
+
+            @test begin
+                g = Rx(1, θ = π)
+                Matrix(g) ≈ [0 -1im; -1im 0]
+            end
+        end
```

Comment:
You forgot to add a newline here.
```suggestion
        end

```